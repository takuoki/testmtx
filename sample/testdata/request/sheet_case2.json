{
  "num_key": 102,
  "string_key": "string value 201",
  "object_key": {
    "key1": 202,
    "key2": "string value 202"
  }
}