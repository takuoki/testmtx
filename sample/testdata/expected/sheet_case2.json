{
  "status": "failure",
  "code": 401
}